src/storage/storage.cc

```diff
@@ -337,15 +337,26 @@ Status Storage::Open(DBOpenMode mode) {
   propagate_opts.disable_auto_compactions = config_->rocks_db.disable_auto_compactions;
   SetBlobDB(&propagate_opts);
 
+  rocksdb::ColumnFamilyOptions lite_opts(subkey_opts);
+  if (config_->lite_mode) {
+    lite_opts.write_buffer_size = 1L;
```

Comment:
We can keep the write_buffer_size as a relatively smaller one like 16MiB/32MiB instead of 1 byte.

Reply:
Yes, It seems that the min value is 16KiB ? Even though it is set to 1, the rocksdb log still shows `Options.write_buffer_size: 65536`.

Reply:
16KiB is good for me.

Reply:
done